### DISCUSSION TITLE
Soft Body Vectorization

### DISCUSSION THREAD

**Discussion by nicholscrawford** [haosulab/ManiSkill#753] [General]
Any updates on vectorized soft body tasks/support? It seems like PhysX supports soft bodies/particle systems, and Sapein is built on PhysX. Is it just a matter of adding tasks? Is there room for contribution in adding support?

**Comment by StoneT2000** [comment]
Let me get back to you with our timeline on soft-body sim stuff. Our team is actually discussing about it a bit more right after new years, we finally got around to making some more actionable plans (beyond referring everyone to ManiSkill2 for soft body sim)

**Reply by yu-mei** [reply]
Thank you, Stone. Is there any new updates on this topic?

**Reply by StoneT2000** [reply]
Slowly being worked on, hopefully by end of summer
